src/supervisor.sbvr

```diff
@@ -0,0 +1,461 @@
+Vocabulary: supervisor
+
+-- "(Type)" refers to the target database type to be autogenerated
+
+-- GENERIC TERMS
+
+Term: download progress
+	Concept Type: Integer (Type)
+
+Term: ip address
+	Concept Type: Short Text (Type)
+
+Term: name
+	Concept Type: Short Text (Type)
+
+Term: port
+	Concept Type: Integer (Type)
+
+Term: slug
+	Concept Type: Short Text (Type)
+
+Term: status
+	Concept Type: Short Text (Type)
+
+Term: version
```

Comment:
Same, is this meant to be a generic version term? Or is it a specific version (e.g. for the supervisor)